src/amber/cli/commands/database.cr

```diff
@@ -89,7 +87,7 @@ module Amber::CLI
           Micrate::DB.connection_url = url.gsub(path, "/#{uri.scheme}")
           return path.gsub("/", "")
         else
-          raise "could not determine database name"
+          raise "Could not determine database name"
```

Comment:
Use exit here as well

Reply:
This is within the scope of the main `rescue` in run. Raising here is more object oriented as exiting is a way that we handle exceptions, not a replacement for exceptions.